pkg/tagger/utils/expiry.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package utils
+
+import (
+	"errors"
+	"sync"
+	"time"
+)
+
+// Expire regularly pools the source keeping the elements reporting.
+// Currently only used for ECSCollector and active containers.
```

Comment:
No need to talk about ECSCollector, or collectors. That could be used by anything.